networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -1,6 +1,6 @@
 type Mutation {
   createGameSessionFromTemplate(input: CreateGameSessionFromTemplateInput!): String 
-  @function(name: "createGame-${env}") 
+  @function(name: "createGameTest-${env}") 
```

Comment:
Amplify issue that I will address when I merge all of these branches in (final name will be `createGame`